setup.cfg

```diff
@@ -51,6 +51,8 @@ install_requires =
 asdf =
   asdf-astropy>=0.1.1
   asdf>=2.8.0
+coordinates =
+  requests
```

Comment:
Do we want to make this a core dep, because at the moment I consider coordinates as one of the main things (if not *the* thing) which you get without installing any extras.

Reply:
Done